src/Renderer/HtmlRenderer.php

```diff
@@ -254,18 +263,39 @@ public function renderCallStack(Throwable $t, array $trace = []): string
             $class = !empty($trace[$i]['class']) ? $trace[$i]['class'] : null;
             $args = !empty($trace[$i]['args']) ? $trace[$i]['args'] : [];
 
+            $parameters = [];
             $function = null;
             if (!empty($trace[$i]['function']) && $trace[$i]['function'] !== 'unknown') {
                 $function = $trace[$i]['function'];
+                if ($class !== null) {
+                    $parameters = (new \ReflectionMethod($class, $function))->getParameters();
+                }
             }
             $index = $i + 2;
 
-            if ($isVendor = $this->isVendorFile($file)) {
-                $vendor[$index] = $this->renderCallStackItem($file, $line, $class, $function, $args, $index, $isVendor);
-                continue;
+            if ($this->isVendorFile($file)) {
+                $vendor[$index] = $this->renderCallStackItem(
+                    $file,
+                    $line,
+                    $class,
+                    $function,
+                    $args,
+                    $index,
+                    true,
+                    $parameters,
+                );
+            } else {
+                $application[$index] = $this->renderCallStackItem(
+                    $file,
+                    $line,
+                    $class,
+                    $function,
+                    $args,
+                    $index,
+                    false,
+                    $parameters,
+                );
```

Comment:
```suggestion
            $vendor[$index] = $this->renderCallStackItem(
                $file,
                $line,
                $class,
                $function,
                $args,
                $index,
                $this->isVendorFile($file),
                $parameters,
            );       
```

Reply:
There two different arrays in use: `$application[$index]` and `$vendor[$index]`